Parsers/ASimAuthentication/Parsers/ASimAuthenticationPostgreSQL.yaml

```diff
@@ -15,160 +15,173 @@ References:
 Description: |
   This ASIM parser supports normalizing PostgreSQL sign in logs to the ASIM Authentication schema.
 ParserName: ASimAuthenticationPostgreSQL
+EquivalentBuiltInParser: _ASim_Authentication_PostgreSQL
```

Comment:
@KustoKing Are these changes due to positions changing, is there anything added new or got removed?
Other than EquivalentBuiltInParser property?

Reply:
Sorted columns and added EventUid